python/chemiscope/input.py

```diff
@@ -590,6 +591,10 @@ def quick_settings(
         view trajectory data: fixing the viewpoint for the structure, reducing the delay
         when cycling between structures and adding a line joining the points in the map.
 
+    :param bool periodic: A boolean flag that sets some default options suitable to
+        view periodic structures: showing the unit cell, and replicating the structure
+         to show atoms at the boundaries of the unit cell.
+
```

Comment:
Instead of doing it in quick setting, which not everyone is using, I was more thinking this should be fixed in the JS code. Whenever we are looking at environments, the supercell should be 3x3x3, unless changed by the user.

Reply:
I don't disagree but that bug report has been open for years, and I think fixing this (or maybe better say "changing this this way") is non-trivial and really a corner case. I think that having sane defaults when generating the output is at least a step forward, and if someone will feel strongly about this in the future they can reopen and figure out how to do it JS-side